test/e2e/server/http_test.go

```diff
@@ -0,0 +1,20 @@
+package server_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/tidwall/gjson"
+
+	"github.com/api7/etcd-adapter/test/e2e/base"
+)
+
+var _ = Describe("HTTP", func() {
+	It("Get version", func() {
+		resp, err := base.GetHTTPClient().R().Get("http://" + base.DefaultAddress + "/version")
+		Expect(err).To(BeNil())
```

Comment:
```suggestion
var _ = Describe("HTTP", func() {
	  It("Get version", func() {
		  resp, err := base.GetHTTPClient().R().Get("http://" + base.DefaultAddress + "/version")
		  Expect(err).To(BeNil())
```

Is it possible to leave spaces in line breaks? Keeping the same indent looks strange...

Reply:
@zaunist This is the automatic code formatting performed by GoLand and gofmt. Should we maybe not modify it manually?

Reply:
Make sense